MaxText/experimental/rl/grpo_inference.yml

```diff
@@ -22,3 +22,19 @@ return_log_prob: True
 
 add_bos: False
 add_eos: False
+
+### Splash attention block sizes
+# These can be tuned for specific hardware generations, and can be set up to
```

Comment:
same question here, plus would be good to point out any differences from grpo.yml settings or guidelines if we have them